dicom-event-driven-ingestion/src/main/java/org/alvearie/imaging/ingestion/RetrieveService.java

```diff
@@ -21,6 +21,7 @@
 import org.alvearie.imaging.ingestion.model.result.DicomEntityResult;
 import org.alvearie.imaging.ingestion.model.result.DicomQueryModel;
 import org.alvearie.imaging.ingestion.model.result.DicomResource;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
Not sure why this came in.  Is it used?

Reply:
Yes, at https://github.com/Alvearie/imaging-ingestion/pull/41/files#diff-3942c379cf1a52c1053b64dcd63beefe72aa48bc5a043279322a69f96199379bR152

It's an existing dependency from quarkus